paywall/src/data-iframe/cache/localStorage.js

```diff
@@ -187,13 +162,14 @@ export async function addListener({
   accountAddress = nullAccount,
 }) {
   const key = storageId(networkId, accountAddress)
```

Comment:
this is key: because we can have storage ids like:

```
unlock-protocol/1/0x....#thing1
unlock-protocol/1/0x....#thing2
```

we can't just assert on `unlock-protocol/1/0x....` because that won't match any of the above examples. The obvious solution is to match on the beginning of the localStorage key, which is `unlock-protocol/1/0x....`